src/themes/mainTheme.ts

```diff
@@ -1,70 +1,72 @@
-import { createTheme, PaletteColorOptions, SxProps } from '@mui/material';
+import { createTheme, darken, PaletteColor, PaletteColorOptions, SxProps } from '@mui/material';
 import { enUS as coreEnUs, nbNO as coreNbNo, nnNO as coreNnNo } from '@mui/material/locale';
 import type {} from '@mui/x-date-pickers/themeAugmentation';
 import i18n from '../translations/i18n';
+import '@mui/material/Chip';
 
 // Colors: https://www.figma.com/file/3hggk6SX2ca81U8kwaZKFs/Farger-NVA
 enum Color {
-  Black = '#222',
+  Black = '#222222',
+  White = '#FFFFFF',
+  PurpleDark = '#0A0132',
+  Purple = '#7351FB',
   CentralImportMain = '#D9D9D9',
-  CentralImportLight = '#EFEEED',
-  ErrorMain = '#AC0303',
-  PrimaryMain = '#0F0035',
-  SecondaryLight = '#F9F4E6',
-  SecondaryMain = '#EDE2C7',
-  SecondaryDark = '#D3C9AF',
-  SuccessMain = '#025810',
-  InfoMain = '#0B408F',
-  InfoLight = '#C2D3EA',
-  PrimaryLight = '#0D4DAD',
+  PurpleLight = '#D1CDFF',
   TextPrimary = 'rgba(0, 0, 0, 0.87)',
-  White = '#fff',
-  DoiRequest = '#FFAA8E',
-  DoiRequestLight = '#FFE2DA',
-  GeneralSupportCase = '#7E9DCC',
-  Registration = '#DAC48E',
-  Person = '#B3D6D9',
-  Project = '#E48F8F',
-  PublishingRequest = '#FFD27B',
-  PublishingRequestLight = '#FFF0D3',
-  NviMain = '#EE95EA',
-  NviLight = '#F8D3F6',
-  WarningMain = '#ED6C02',
-  WarningLight = '#FFD45A',
+  Registration = '#EED2AE',
+  Person = '#BAD2F7',
+  Project = '#E8B8D0',
+  Outdated = '#fc53db',
+  Neutral97 = '#F7F7F7',
+  Neutral95 = '#F1F1F1',
+  Neutral87 = '#DEDEDE',
+  Neutral46 = '#767676',
+  Blue = '#004FCF',
+  BlueLight = '#E6F0FF',
+  Green = '#096638',
+  GreenLight = '#CFF7E2',
+  Red = '#B60203',
+  RedLight = '#FFEAE9',
+  Yellow = '#FFB700',
+  YellowLight = '#FCEED2',
 }
 
 const coreLocale = i18n.language === 'eng' ? coreEnUs : i18n.language === 'nno' ? coreNnNo : coreNbNo;
 
 declare module '@mui/material/styles' {
   interface Palette {
-    registration: PaletteColorOptions;
-    person: PaletteColorOptions;
-    project: PaletteColorOptions;
-    publishingRequest: PaletteColorOptions;
-    doiRequest: PaletteColorOptions;
-    generalSupportCase: PaletteColorOptions;
-    centralImport: PaletteColorOptions;
-    nvi: PaletteColorOptions;
+    tertiary: PaletteColor;
+    registration: PaletteColor;
+    person: PaletteColor;
+    project: PaletteColor;
+    centralImport: PaletteColor;
   }
   interface PaletteOptions {
+    tertiary: PaletteColorOptions;
     registration?: PaletteColorOptions;
     person?: PaletteColorOptions;
     project?: PaletteColorOptions;
-    publishingRequest?: PaletteColorOptions;
-    doiRequest?: PaletteColorOptions;
-    generalSupportCase?: PaletteColorOptions;
     centralImport?: PaletteColorOptions;
-    nvi?: PaletteColorOptions;
+  }
+  interface TypeBackground {
+    neutral97?: string;
+    neutral95?: string;
+    neutral87?: string;
+    neutral46?: string;
```

Comment:
Sliter veldig med å skjønne disse altså, men vet du har hatt noen runder på det, så la gå..

Reply:
er veldig for å endre navnet på disse, men klarte ikke å bli enig med meg selv om hva det skulle være. Tenkte på `level`, `ome. two, three`, eller bare oppdatere de fargene vi har satt til `grey` i theme i dag, da de ikke brukes så mange plasser.